src/report/graphs.py

```diff
@@ -491,7 +491,7 @@ def too_close(self, trips):
         """Determine if we are too close."""
```

Comment:
I'll approve but I IMHO this heredoc could be better.

Reply:
By heredoc do you mean pydoc?  If so then I agree that the documentation for these files is garbage.  This is ancient code.

Reply:
@jsf9k Yes, pydoc :)

Reply:
@jasonodoom - Can we resolve this conversation?